ospfd/ospf_lsa.c

```diff
@@ -2269,6 +2269,10 @@ struct ospf_lsa *ospf_external_lsa_refresh(struct ospf *ospf,
 		return NULL;
 	}
 
+	if (CHECK_FLAG(lsa->flags, OSPF_LSA_IN_MAXAGE)) {
+		ospf_lsa_maxage_delete(ospf, lsa);
+	}
```

Comment:
LSA is being referenced down in function below. If ospf_lsa_maxage_delete frees lsa then we will be access freed pointer.

Reply:
This check is removed. It is not required. 
Thanks,